awsconfigs/common/istio-ingress/overlays/https/params.env

```diff
@@ -0,0 +1 @@
+certArn=
```

Comment:
Can we add a comment here on purpose of this variable? 
I understand the README.md will have the details but adding a line here doesn't hurt.

Reply:
I would also suggest to create another overlay for http.